src/FSLibrary/MissionSorobanInvokeHostLoad.fs

```diff
@@ -36,5 +36,29 @@ let sorobanInvokeHostLoad (context: MissionContext) =
             formation.UpgradeMaxTxSetSize [ coreSet ] 100000
 
             formation.RunLoadgen coreSet context.GenerateAccountCreationLoad
-            formation.RunLoadgen coreSet context.SetupSorobanInvoke
-            formation.RunLoadgen coreSet context.GenerateSorobanInvokeLoad)
+
+            // Upgrade to overly pessimistic Phase 1 limits, where TXs have phase 1 limits
+            // and ledger limits are unbounded
+            formation.SetupUpgradeContract coreSet
+
+            formation.DeployUpgradeEntriesAndArm
+                coreSet
+                [ coreSet ]
+                { LoadGen.GetSorobanPhase1Upgrade() with
+                      ledgerMaxInstructions = Some(100_000_000L * 30L)
+                      ledgerMaxReadBytes = Some(133_120 * 30) // 130 KB * 30
+                      ledgerMaxWriteBytes = Some(66_560 * 30) // 65 KB * 30
+                      ledgerMaxReadLedgerEntries = Some(30 * 30)
+                      ledgerMaxWriteLedgerEntries = Some(20 * 30) }
+                (System.DateTime.UtcNow.AddSeconds(20.0))
```

Comment:
same thing here re:upgrades, need to arm to upgrade asap and wait until it goes through.